tv.anilibria.anilibria.yml

```diff
@@ -0,0 +1,29 @@
+app-id: tv.anilibria.anilibria
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: /app/bin/AniLibria
+modules:
+  - name: anilibria
+    buildsystem: simple
```

Comment:
why not using buildsystem: qmake ?

Reply:
the project is not stored in the root folder and qmake cannot find the pro file.

Reply:
there is `dir` to tell where the top source level is.

Reply:
From the man:

> subdir (string)
>           Build inside this subdirectory of the extracted sources